inlang/documentation/ecosystem/overview/architecture.md

```diff
@@ -1,29 +1,19 @@
 # Architecture
 
-Inlang's architecture consists of four main components:
+Inlang's architecture consists of three main components:
 
-1. [Applications](/documentation/concept/app) - Provide users with an interface for their projects.
-2. [SDK](/documentation) - Can be used by developers to build inlang apps.
-3. [Project file](/documentation/concept/project) - Defines the project's settings, which language tags exists and to be imported modules.
-4. [Modules](/documentation/plugin) - Contain plugins and/or lint rules that define the behavior of a project.
+1. [inlang Apps](/documentation/concept/app) - Provide users with an interface for their projects.
```

Comment:
"apps" feels overused - e.g. app developers think of apps as the apps which they own. Unfortunately I don't have a better word.

It's easy to confuse "inlang apps" as "my app projects that use inlang" instead of "all the tools available for working with inlang projects"

How about:
> [inlang Apps](/documentation/concept/app) are all the tools available for working with inlang projects.